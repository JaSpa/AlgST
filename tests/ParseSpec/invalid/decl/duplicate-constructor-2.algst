data Where = Here | Here Int
