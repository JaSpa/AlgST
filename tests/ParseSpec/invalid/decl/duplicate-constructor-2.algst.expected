
1:21: error:
	 Multiple declarations of Here
	 Declared at: 1:14
	              1:21